Src/WitsmlExplorer.Frontend/components/Modals/ServerModal.tsx

```diff
@@ -1,50 +1,104 @@
-import { Autocomplete } from "@equinor/eds-core-react";
-import { Button, TextField } from "@material-ui/core";
-import MuiThumbUpOutlinedIcon from "@material-ui/icons/ThumbUpOutlined";
-import React, { ChangeEvent, useState } from "react";
+import { Autocomplete, Button, EdsProvider, Label, TextField } from "@equinor/eds-core-react";
```

Comment:
It seems to me that all the changes to the ServerModal relate to the #1590 issue. Is it necessary to do it together with #1589? Even then, as [commented by Marita in 1590](https://github.com/equinor/witsml-explorer/issues/1590#issuecomment-1296993056), username and password fields should not be in the server modal. This means handling credentials in lines 66-100 is not expected.

Reply:
updated!